python/cuml/tests/test_random_forest.py

```diff
@@ -307,6 +307,67 @@ def test_rf_classification(small_clf, datatype, max_samples, max_features):
     assert fil_acc >= (cuml_acc - 0.07)  # to be changed to 0.02. see issue #3910: https://github.com/rapidsai/cuml/issues/3910 # noqa
 
 
+@pytest.mark.parametrize(
+    "max_samples", [unit_param(1.0), quality_param(0.90), stress_param(0.95)]
+)
+@pytest.mark.parametrize("datatype", [np.float32, np.float64])
+@pytest.mark.parametrize("max_features", [1.0, "auto", "log2", "sqrt"])
+@pytest.mark.parametrize("b", [0, 5, -5, 10])
+@pytest.mark.parametrize("a", [1, 2, 3])
```

Comment:
I don't think we need this full matrix of tests for testing the monotonic case, one combination for each datatype would be enough?

Reply:
Sure, I will fix one value for a, b and max_features.